### PR TITLE
Feat/position cell UI

### PR DESCRIPTION
**Pull request by baikov98**

### CODE REVIEW

File: packages/app/src/entities/position/context/provider.tsx [hunk 1]
```diff
@@ -66,12 +66,10 @@ export function PositionProvider({ children }: { children: JSX.Element }) {
       [],
     )
 
-    if (lendPositions.length || debtPositions.length) {
-      setPosition({
-        deposits: lendPositions,
-        debts: debtPositions,
-      })
-    }
+    setPosition({
```
Comment (Artem1211) [inline packages/app/src/entities/position/context/provider.tsx:69]: If your lendPositions.length and debtPositions.length are equal to 0 you need setPosition(undefined)

Comment (baikov98) [reply packages/app/src/entities/position/context/provider.tsx]: fixed

File: packages/app/src/widgets/position-section/components/position-cell/position-cell.tsx [hunk 2]
```diff
@@ -25,36 +27,54 @@ export function PositionCell({
...
+          <Typography caption>
+            {tokenCache?.name} {percentage && percentage !== 100 ? `· ${percentage}%` : null}
+          </Typography>
+          <S.PositionCellTokenAmount>
+            <Typography>
+              {value.toString(10)} {tokenCache?.symbol}{' '}
+            </Typography>
+            {interestRate && (
+              <Label positive={isLendPosition} negative={!isLendPosition} sm>
+                {isLendPosition ? '+' : '-'}
```
Comment (bouvens) [inline packages/app/src/widgets/position-section/components/position-cell/position-cell.tsx:55]:
```suggestion
                {isLendPosition ? '+' : '&minus;'}
```

File: packages/app/src/widgets/position-section/position-section.tsx [hunk 3]
```diff
@@ -64,104 +57,100 @@ export function PositionSection() {
...
+                  percentage={depositPercentage}
+                  openDecreaseModal={() => openLendDecreaseModal(tokenName)}
+                  openIncreaseModal={() => openLendIncreaseModal(tokenName)}
+                  isLendPosition
+                />
+              )
+            })}
+            {!isFullPosition && (
+              <button type="button" onClick={() => openLendIncreaseModal()}>
+                + lend more
```
Comment (bouvens) [inline packages/app/src/widgets/position-section/position-section.tsx:98]:
```suggestion
                + Lend More
```

File: packages/app/src/widgets/position-section/position-section.tsx [hunk 4]
```diff
@@ -64,104 +57,100 @@ export function PositionSection() {
...
+                </Typography>
+              </S.PositionSumWrapper>
+            ) : (
+              <>
+                <S.PositionSumWrapper>
+                  {debtsSumUsd && <Typography headerL>{formatUsd(debtsSumUsd)}</Typography>}
+                  <Typography>Borrowed</Typography>
+                </S.PositionSumWrapper>
+                <Label negative lg>
+                  -{formatPercent(debtSumInterestRate)} APR
```
Comment (bouvens) [inline packages/app/src/widgets/position-section/position-section.tsx:119]:
```suggestion
                  &minus;{formatPercent(debtSumInterestRate)} APR
```

File: packages/app/src/widgets/position-section/position-section.tsx [hunk 5]
```diff
@@ -64,104 +57,100 @@ export function PositionSection() {
...
+                  position={debt}
+                  percentage={debtPercentage}
+                  openDecreaseModal={() => openBorrowDecreaseModal(tokenName)}
+                  openIncreaseModal={() => openBorrowIncreaseModal(tokenName)}
+                />
+              )
+            })}
+            {!isFullDeposits && (
+              <button type="button" onClick={() => openBorrowIncreaseModal()}>
+                + lend more
```
Comment (bouvens) [inline packages/app/src/widgets/position-section/position-section.tsx:144]:
```suggestion
                + Lend More
```

**Review by Artem1211 — APPROVED** [review]

**Review by bouvens — APPROVED** [review]

File: packages/app/src/widgets/position-section/components/position-cell/position-cell.tsx (hunk 2 above)
Comment (baikov98) [reply packages/app/src/widgets/position-section/components/position-cell/position-cell.tsx]: fixed

File: packages/app/src/widgets/position-section/position-section.tsx (hunk 3 above)
Comment (baikov98) [reply packages/app/src/widgets/position-section/position-section.tsx]: fixed

File: packages/app/src/widgets/position-section/position-section.tsx (hunk 4 above)
Comment (baikov98) [reply packages/app/src/widgets/position-section/position-section.tsx]: fixed

File: packages/app/src/widgets/position-section/position-section.tsx (hunk 5 above)
Comment (baikov98) [reply packages/app/src/widgets/position-section/position-section.tsx]: fixed

**Review by bouvens — APPROVED** [review]

**Pull request merged**